proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -36,14 +36,14 @@ message MsgTransfer {
   // the channel by which the packet will be sent
   string source_channel = 2;
   // the tokens to be transferred
-  cosmos.base.v1beta1.Coin token = 3 [(gogoproto.nullable) = false, (amino.encoding) = "legacy_coin"];
```

Comment:
`(amino.encoding) = "legacy_coin"` instructions the amino codec on how to encode `nil` slices of `Coin` - you can see that here -> https://github.com/cosmos/cosmos-sdk/blob/main/x/tx/signing/aminojson/json_marshal.go#L70